src/scripts/update_funding_round.js

```diff
@@ -51,6 +52,7 @@ const main = async () => {
         curRoundVoteEnd = curRound.get('Voting Ends')
         curRoundVoteType = curRound.get('Vote Type')
         curRoundBallotType = curRound.get('Ballot Type')
+        await processAirtableProposalStandings(curRoundNumber) // process proposal standings
```

Comment:
Yeah, I was going to comment on this too, but my comment is a bit different... 

1. This block is mostly to initialize the data (might be good inside a try/catch block).
2. processAirtableProposalStandings() needs to be called regardless of what's happening in the state, because proposals are getting updated manually, and will eventually also be updated via Pearl.

So, imagine the scenario where we receive a bunch of proposal updates via port, that their project has completed their deliverables.
1. Someone would enter that data manually into airtable
2. DAOBot would then update that project's standings 

So, it's not just when prepareProposalsForSnapshot, because that only happens at a very specific time frame.
These updates need to be happening often, so the state is always up-to-date based on the latest information it has.

Hope this comment helps!
Thank you!